.devcontainer/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM ruby:2.6.3-buster
+
+# Avoid warnings by switching to noninteractive
+ENV DEBIAN_FRONTEND=noninteractive
+
+# The ruby image comes with a base non-root 'ruby' user which this Dockerfile
+# gives sudo access. Hoewver, for Linux, this user's GID/UID must match your local
+# user UID/GID to avoid permission issues with bind mounts. Update USER_UID / USER_GID
+# if yours is not 1000. See https://aka.ms/vscode-remote/containers/non-root-user.
+ARG USERNAME=ruby
+ARG USER_UID=1000
+ARG USER_GID=$USER_UID
+
+# Configure apt and install packages
+RUN apt-get update \
```

Comment:
The continuations is a Docker thing. Every run command adds a layer to the image as a file system union making the final image larger and slightly less efficient. They might have improved things since then but I'm not sure. This is still how I see most Docker files written.

Reply:
ok, I hate it but that sounds like a personal problem

Reply:
Even official docker images still do the same thing, https://github.com/docker-library/python/blob/0581bee3d9811695d854ea0e863781df67fbd390/3.8-rc/buster/Dockerfile

So yeah, just a personal thing 😄